src/tree_schema.h

```diff
@@ -1857,13 +1859,35 @@ const struct lysc_notif *lysc_node_notifs(const struct lysc_node *node);
 
 /**
  * @brief Get the children linked list of the given (compiled) schema node.
- * Decides the node's type and in case it has a children list, returns it.
+ * Skips over input and output nodes.
```

Comment:
please refer alternative lysc_node_children_all() and vice versa

Reply:
Fixed.